### ISSUE TITLE
Diffexp default cluster

### ISSUE THREAD

**Issue by AlexanderPico** [RBVI/scNetViz#7]
With the following commands, I can step through from experiment to networks (cool!)

```
cmd.str <- 'scnetviz list gxa entries species="Homo sapiens"'
exp.list2 <- commandsPOST(cmd.str)
exp <- exp.list2[[1]]$accession
exp.load <- commandsRun(paste0('scnetviz load gxa experiment',
                               ' accession=', exp, 
                               ' showTable=false'))
commandsPOST(paste0('scnetviz calculate diffexp',
                   ' accession=', exp,
                   ' category=Cluster',
                   ' categoryRow=4'))
commandsRun(paste0('scnetviz create network',
                   ' accession=', exp))

```

However, it would be nice if I could rely on additional defaults for `calculate diff exp`. I couldn't get it to run without ' category=Cluster' and 'categoryRow=4', for example. Is there a way to set these defaults for the automation use case? Otherwise, this requires the user to open the categories table, make some decisions and manually enter in some values.

**Comment by RBVI** [comment]
Definitely should be able to get at the defaults.  I'll try to fix this and #8 today.

**Comment by RBVI** [comment]
Fixed.
